tests/helpers/lib.sh

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env bash
```

Comment:
Generic bash helpers, not specific to nydus (provides logging, http, heathcheck, minimal github client).